docs/core/docker/build-container.md

```diff
@@ -22,58 +22,71 @@ You'll learn to:
 
 You'll understand the Docker container build and deploy tasks for a .NET Core application. The *Docker platform* uses the *Docker engine* to quickly build and package apps as *Docker images*. These images are written in the *Dockerfile* format to be deployed and run in a layered container.
 
-> [!WARNING]
-> **This tutorial isn't for ASP.NET Core apps.** If you're using ASP.NET Core, read the [Learn how to containerize an ASP.NET Core application](/aspnet/core/host-and-deploy/docker/building-net-docker-images) tutorial.
+> [!NOTE]
+> This tutorial **is not** for ASP.NET Core apps. If you're using ASP.NET Core, see the [Learn how to containerize an ASP.NET Core application](/aspnet/core/host-and-deploy/docker/building-net-docker-images) tutorial.
 
 ## Prerequisites
 
 Install the following prerequisites:
 
 - [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download)\
 If you have .NET Core installed, use the `dotnet --info` command to determine which SDK you're using.
-
 - [Docker Community Edition](https://www.docker.com/products/docker-desktop)
-
 - A temporary working folder for the *Dockerfile* and .NET Core example app. In this tutorial, the name *docker-working* is used as the working folder.
 
 ## Create .NET Core app
 
 You need a .NET Core app that the Docker container will run. Open your terminal, create a working folder if you haven't already, and enter it. In the working folder, run the following command to create a new project in a subdirectory named *app*:
 
 ```dotnetcli
-dotnet new console -o app -n myapp
+dotnet new console -o App -n NetCore.Docker
 ```
 
 Your folder tree will look like the following:
 
 ```
 docker-working
-│
-└───app
-    │   myapp.csproj
-    │   Program.cs
-    │
-    └───obj
-            myapp.csproj.nuget.cache
-            myapp.csproj.nuget.dgspec.json
-            myapp.csproj.nuget.g.props
-            myapp.csproj.nuget.g.targets
-            project.assets.json
+    └──App
+        ├──NetCore.Docker.csproj
+        ├──Program.cs
+        └──obj
+            ├──NetCore.Docker.csproj.nuget.dgspec.json
+            ├──NetCore.Docker.csproj.nuget.g.props
+            ├──NetCore.Docker.csproj.nuget.g.targets
+            ├──project.assets.json
+            └──project.nuget.cache
+```
+
+The `dotnet new` command creates a new folder named *App* and generates a "Hello World" console application. First, changes directories and navigate into the *App* folder.
+
+```dotnetcli
+cd App
 ```
 
-The `dotnet new` command creates a new folder named *app* and generates a "Hello World" app. Enter the *app* folder and run the command `dotnet run`. You'll see the following output:
+Execute the `dotnet run` command. The application will run, and print `Hello World!` below the command:
 
-```console
```

Comment:
This article may not be using the latest styles we've come up with, but currently we use `dotnetcli` only when the only commands run are a dotnet command. Otherwise, it's possible that coloring is wrong. You can kind of see that with the `dontnetcli` codeblock above, `cd` is colored.

So we've adopted using `dotnetcli` when the only command (without output) is the `dotnet` command. If it's a command + output, use `console` or separate them with something like:

> Execute the `dotnet run` command.
> -- Codeblock dotnetcli with dotnet run command
> The application prints "Hello World".
> -- Codeblock console with with hello world printed